KinConnections/resources/email_wrapper.py

```diff
@@ -0,0 +1,96 @@
+# email_wrapper.py - KinConnections
+# Serves as wrapper for sending connection emails
+
+# for environment variables
+import os
+from dotenv import load_dotenv
+# for email engine
+import smtplib, ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.utils import formataddr
+# for rich html email
+from dominate import document
+from dominate.tags import *
+# flask
+from flask import Blueprint, Response, request, render_template, session, redirect, url_for
+from flask_restful import Resource
+
+from .db_wrapper import *
+
+
+# access credentials from .env
+load_dotenv(dotenv_path="../.env")
```

Comment:
Q: Do we have to do this here? Does the context not carry over from `app.py`. It would be nice if we could simply just access the`env` variable here.